imap_processing/tests/ultra/unit/test_ultra_l1b_culling.py

```diff
@@ -40,20 +40,16 @@ def test_data(use_fake_spin_data_for_time):
     return time, spin_number, energy, expected_counts
 
 
-def test_get_spin(use_fake_spin_data_for_time, l1b_datasets):
+def test_get_spin(use_fake_spin_data_for_time):
     """Tests get_spin function."""
 
-    de_dataset = l1b_datasets[0]
-    use_fake_spin_data_for_time(
-        de_dataset["event_times"][0], de_dataset["event_times"][-1]
-    )
-    spin_number = get_spin(de_dataset["event_times"])
+    start = 4.45015658e08
+    stop = 4.45015873e08
```

Comment:
It might be more obvious to add the offset here rather than two large numbers. Additionally, perhaps explicitly set your nspins and spin_period rather than calculating nspins based upon those below. Looking at the test, I'm unclear whether the linspace=20 would be the same number of spins as what you've calculated, so making sure they are different will tell you about the unique being a different length than the npoints.
```suggestion
    nspins = 5
    spin_period = 15
    start = 4.45015658e08
    stop = start + (nspins + 1) * spin_period
```